### ISSUE TITLE
[v1.0.0b] - long running query (postgres)

### ISSUE THREAD

**Issue by ahgraber** [mealie-recipes/mealie#1892]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

I'm receiving long-running query warnings for the following query (I'm using postgres, and I'm aware this is in beta).  The issue is that this process doesn't close and the query state remains `idle in transaction`.

```sql
SELECT groups.id AS groups_id, groups.name AS groups_name, groups.webhook_enable AS groups_webhook_enable, groups.webhook_time AS groups_webhook_time 
FROM groups 
WHERE groups.id = 1 
 LIMIT 1
```

If I run this query manually, it returns successfully and the transaction closes:

```csv
groups_id,groups_name,groups_webhook_enable,groups_webhook_time
1,Home,false,"00:00"
```

### Deployment

Docker (Linux)

### Deployment Details

I'm running the linux container image [using kubernetes (k3s)](https://github.com/ahgraber/homelab-gitops-k3s/tree/main/cluster/apps/services/mealie), backed by a postgres database

**Comment by hay-kot** [comment]
Can you provide all the version information from your settings page? I think we've fixed this already, but I want to make sure.

**Comment by ahgraber** [comment]
Sure -- 

here's a screenshot of the settings page:
![Screen Shot 2022-12-13 at 14 57 08](https://user-images.githubusercontent.com/24922003/207481366-0afddb1d-d38c-4ad5-8b36-1e1f25afa7cc.jpg)

here's the tag and sha of the image I'm pulling:

* docker.io/hkotel/mealie:frontend-v1.0.0beta-5
* docker.io/hkotel/mealie@sha256:d8ff0ecd5cfe460fb593a99fa78d9ca17401925d287c723a319abd764f80b9f7

**Comment by hay-kot** [comment]
@ahgraber

Apologies for the late reply. I'm not really sure what you're running, but it isn't the latest Beta. Please see the migration guide for moving to the latest beta where this issue should be resolved 

https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/

**Comment by ahgraber** [comment]
Thanks.  I updated to a nightly build, pinning the image digest and the long-running query issue seems to be resolved.
However, I couldn't get my postgres setup to work .. I kept getting barman errors when running the WAL archive.   Does mealie's automatic backup work with postgres as well? 

In any case, postgres is probably overbuild for my use case, so I switched to sqlite and everything is running smoothly.  I'll close this issue.
